vochain/vochaininfo/vochaininfo.go

```diff
@@ -128,21 +128,21 @@ func (vi *VochainInfo) VoteCacheSize() int {
 	return vi.voteCacheSize
 }
 
-// netInfo returns network info (mainly, peers)
-func (vi *VochainInfo) netInfo() (*coretypes.ResultNetInfo, error) {
+// NetInfo returns network info (mainly, peers)
+func (vi *VochainInfo) NetInfo() (*coretypes.ResultNetInfo, error) {
 	if vi.vnode.Node != nil {
 		return vi.vnode.Node.NetInfo(context.Background())
 	}
 	return nil, fmt.Errorf("vochain node not ready")
 }
 
 // Peers returns the whole list of peers, including ID and URL
-func (vi *VochainInfo) Peers() []coretypes.Peer {
-	ni, err := vi.netInfo()
+func (vi *VochainInfo) Peers() int {
```

Comment:
maybe call it NPeers then? to make it consistent with what's return'ed

Reply:
yep